assets/js/modules/analytics-4/index.js

```diff
@@ -695,6 +695,13 @@ export const registerNotifications = ( notifications ) => {
 				areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
 				viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
 				checkRequirements: async ( { select, resolveSelect } ) => {
+					const analyticsConnected =
+						select( CORE_MODULES ).isModuleConnected();
```

Comment:
No slug is passed here so it will always return `false`. I'll open a new issue to address this as it should raise an error when doing this.